src/main/java/org/embulk/output/mailchimp/MailChimpClient.java

```diff
@@ -219,7 +233,7 @@ public String apply(CategoriesResponse input)
             List<MergeField> allMergeFields = new ArrayList<>();
```

Comment:
This method may also have an issue with pagination?

Reply:
Arg, thanks!